pyflagser/tests/test_flagser.py

```diff
@@ -178,20 +178,20 @@ def are_matrices_equal(m1, m2):
 def test_betti(flag_file):
     betti_exp = betti[os.path.split(flag_file)[1]]
     flag_matrix = loadflag(flag_file, fmt='coo')
-    betti_res = flagser(flag_matrix)["betti"]
+    betti_res = flagser_persistence(flag_matrix)["betti"]
     assert_almost_equal(betti_res, betti_exp)
 
 
-def test_filtrations_d5(flag_file, filtration):
-    """Testing all filtrations available for dataset d5.flag,
-    see conftest.py"""
-    flag_matrix = loadflag(flag_file, fmt='coo')
-    res = flagser(flag_matrix, max_dimension=1, directed=False,
-                  filtration=filtration)
-    for filt, tests in filtrations_results.items():
-        if filtration == filt:
-            tmp = np.array(res["dgms"]).tolist()
-            tmp2 = np.array(tests["dgms"]).tolist()
-            assert are_matrices_equal(tmp, tmp2), \
-                "Diagrams {} \n and {} \n are not equal"\
-                .format(tmp, tmp2)
+# def test_filtrations_d5(flag_file, filtration):
```

Comment:
Are you planning to remove these comments before we merge?

Reply:
Yes, done!